### DISCUSSION TITLE
ThemeSwitcher and SecureUxTheme

### DISCUSSION THREAD

**Discussion by WildByDesign** [OrthodoxWindows/ThemeSwitcherScripts#7] [General]
I came across your repo and found it quite useful. I just wanted to point out one thing on your Readme. SecureUxTheme does work well with ThemeSwitcher and I've used the combo quite a bit recently. You just have to add the hook for ThemeSwitcher.exe in the registry, similar to how SecureUxTheme hooks winlogon.exe and others.

I'm attaching the registry file here but I had to add .txt extension for Github to accept it. You just have to remove that.
[ThemeSwitcher.exe.reg.txt](https://github.com/user-attachments/files/16118936/ThemeSwitcher.exe.reg.txt)

Thank you for sharing your great scripts.

**Comment by OrthodoxWindows** [comment]
Hello WildByDesign,
Thanks for the advice, I will try it soon. Sometimes we forget things that are next to us, that's what I did.

**Comment by OrthodoxWindows** [comment]
Concerning ThemeSwticherScript, I have improved quite a few things, as I have at all I will be able to perfect these improvements.
Thank you for your interest in the project.

**Reply by WildByDesign** [reply]
You're welcome.
